deploy/charts/trust-manager/values.yaml

```diff
@@ -182,6 +182,13 @@ app:
   # The verbosity of trust-manager logging. This takes a value from 1-5, with the higher value being more verbose.
   logLevel: 1
 
+  # Leader Election Configuration
+  leaderElection:
+    # How long will a lease be valid? You probably want to increase this to 60-120s.
```

Comment:
```suggestion
    # The duration that non-leader candidates will wait to force acquire leadership.
    # The default should be sufficient in a healthy cluster but can be slightly increased to prevent trust-manager from restart-looping when the API server is overloaded.
```